### ISSUE TITLE
Only get 16% accuracy from this example.

### ISSUE THREAD

**Issue by JeremyFxx** [iwyoo/ELM-tensorflow#3]
HI, iwyoo.

Your code is very helpful for my work. I test your code on my machine, but I only get 16% instead of the reported 84%. The log is as follows.

/home/jeremyfeng/.virtualenvs/tensorflow/bin/python /home/jeremyfeng/Dropbox/PycharmProjects/ELM_tensorflow_iwyoo/mnist_example.py
2018-01-22 11:56:41.815772: I tensorflow/core/platform/cpu_feature_guard.cc:137] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2018-01-22 11:56:41.915479: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:892] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2018-01-22 11:56:41.915717: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1030] Found device 0 with properties: 
name: GeForce GTX 1060 major: 6 minor: 1 memoryClockRate(GHz): 1.733
pciBusID: 0000:01:00.0
totalMemory: 5.92GiB freeMemory: 5.09GiB
2018-01-22 11:56:41.915728: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1120] Creating TensorFlow device (/device:GPU:0) -> (device: 0, name: GeForce GTX 1060, pci bus id: 0000:01:00.0, compute capability: 6.1)
Extracting ./MNIST_data/train-images-idx3-ubyte.gz
Extracting ./MNIST_data/train-labels-idx1-ubyte.gz
Extracting ./MNIST_data/t10k-images-idx3-ubyte.gz
Extracting ./MNIST_data/t10k-labels-idx1-ubyte.gz
batch_size : 5000
hidden_num : 150
WARNING:tensorflow:From /home/jeremyfeng/.virtualenvs/tensorflow/lib/python3.5/site-packages/tensorflow/python/util/tf_should_use.py:107: initialize_variables (from tensorflow.python.ops.variables) is deprecated and will be removed after 2017-03-02.
Instructions for updating:
Use `tf.variables_initializer` instead.
2018-01-22 11:56:44.123492: I tensorflow/core/kernels/cuda_solvers.cc:159] Creating CudaSolver handles for stream 0x4f323a0
Accuracy: 0.160799995

Process finished with exit code 0

So could you help me finding what is wrong with my configure.

Best regards.

Jeremy

**Comment by namedysx** [comment]
Hi iwyoo,
I also only get 9%.  when hidden nodes add to 1000, accuracy just add to 19%.
